### PR TITLE
Add souls as a currency

### PR DESCRIPTION
**Pull request by robmart**
## Changelog:
- Added Souls as a currency. They are gained by killing people.

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
* You can view current amount of souls in the debug decisions (wild_goose_chase_decision, add_soul, remove_soul).
* Try to kill someone (Quick imprison -> execute) and see if the number increments

### CODE REVIEW

File: common/effect_localization/wc_magic_effects.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+REMOVE_SOULS = {
+	global = REMOVE_SOULS
+	first = REMOVE_SOULS
+	third = REMOVE_SOULS
+	
+	global_past = REMOVE_SOULS
+	first_past = REMOVE_SOULS
+	third_past = REMOVE_SOULS
+}
```
Comment (zumbak04) [inline common/effect_localization/wc_magic_effects.txt:28]: You don't need both remove and add effects.
Check `change_dark_frenzy_effect`.

**Review by zumbak04 — COMMENTED** [review]
> Try to kill someone (Quick imprison -> execute) and see if the number increments

It does.

File: common/effect_localization/wc_magic_effects.txt (hunk 1 above)
Comment (robmart) [reply common/effect_localization/wc_magic_effects.txt]: Fixed in 
4a7ec34

File: localization/english/wc_magic_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿l_english:
+ soul_amount:0 "You currently have [GetPlayer.MakeScope.Var('collected_souls').GetValue|0V] Souls."
```
Comment (zumbak04) [inline localization/english/wc_magic_l_english.yml:2]: Vanilla current value is written in a different style.
<details>
<summary>Click to expand</summary>

image

</details>

**Pull request closed without merging**